app/src/constants/requestMethodColors.ts

```diff
@@ -1,9 +1,15 @@
+import { generateColorTokens } from "lib/design-system-v2/tokens/colors";
+
+const tokens = generateColorTokens("#004EEB", "A447F8", "#787878", "#0BAA60", "#E43434", "#E09400");
```

Comment:
@wrongsahil  this mentions to use the `useTheme` hook which can only be accessed inside a react component. 

Here, we need to create a reusable object. that can be imported in multiple components.

Do you think we should rather add [this object](https://github.com/requestly/requestly/blob/master/app/src/constants/requestMethodColors.ts#L5-L13) in someway inside the theme to avoid making this file altogether?

Reply:
No. We shouldn't add this to theme as this is business logic. Theme should be generic.

Reply:
Let me check how to use the theme outside of react components